frameworks/JavaScript/fastify/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "fastify-tfb",
+  "version": "0.0.1",
+  "description": "Fastify tests for TechEmpower Framework Benchmarks.",
+  "main": "app.js",
+  "private": true,
+  "dependencies": {
+    "fastify": "^1.9.0",
+    "handlebars": "~4.0.11",
+    "mongoose": "~5.2.5",
+    "mysql2": "~1.5.3",
+    "pg": "~7.4.1",
+    "pg-hstore": "~2.3.2",
+    "point-of-view": "^1.1.0",
+    "sequelize": "~4.37.3"
```

Comment:
Not sure of the syntax here, but could you lock these to exact versions? A problem we frequently encounter is dependencies causing trouble because someone marks "at least version 1.1.0" and on a much later benchmarking run, the newest version is 1.2.0 and it breaks the implementation.